docs/compression_algorithms/post_training/Quantization.md

```diff
@@ -91,3 +91,22 @@ NNCF provides the examples of Post-Training Quantization where you can find the
 function: [PyTorch](../../../examples/post_training_quantization/torch/mobilenet_v2/README.md), [TensorFlow](../../../examples/post_training_quantization/tensorflow/mobilenet_v2/README.md), [ONNX](../../../examples/post_training_quantization/onnx/mobilenet_v2/README.md), and [OpenVINO](../../../examples/post_training_quantization/openvino/mobilenet_v2/README.md)
 
 In case the Post-Training Quantization algorithm could not reach quality requirements you can fine-tune a quantized pytorch model. Example of the Quantization-Aware training pipeline for a pytorch model could be found [here](../../../examples/quantization_aware_training/torch/resnet18/README.md).
+
+## Dataloader Usage
```

Comment:
```suggestion
## Using `pytorch.Dataloader` or `tf.data.Dataset` as data source for calibration dataset.
```
I think you can add a general information about what kind `dataloader` are supported and how NNCF handle batch size. 

If you want to give only information about only batch size support then you can call topic as following: "Batch size support in calibration dataset."

Reply:
Applied